src/store/ChartStore.js

```diff
@@ -123,6 +123,59 @@ class ChartStore {
         this.chartNode = this.rootNode.querySelector('.ciq-chart-area');
         this.chartControlsNode = this.rootNode.querySelector('.cq-chart-controls');
 
+        CIQ.ChartEngine.prototype.createYAxisLabel = function (panel, txt, y, backgroundColor, color, ctx, yAxis) {
```

Comment:
@shayanbinary I dont think its good to override the whole `createYAxisLabel `, if we cant change the style can you check for a listener or injecttion, something like `headsUpHR` and just change float label inline style there?

Reply:
@MahboobehMohammadi actually by `drawCurrentHR` you just be able to set color and background, but for the new design, we need to handle `radius, height and width` which these props effect directly from this method